src/libraries/System.Numerics.Tensors/ref/System.Numerics.Tensors.netcore.cs

```diff
@@ -157,4 +386,94 @@ public static partial class TensorPrimitives
+        public nint LinearLength { get { throw null; } }
+        public int Rank { get { throw null; } }
+        public System.ReadOnlySpan<nint> Shape { get { throw null; } }
+        public System.ReadOnlySpan<nint> Strides { get { throw null; } }
+        public void Clear() { }
+        public void CopyTo(System.Numerics.Tensors.TensorSpan<T> destination) { }
+        [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
+        [System.ObsoleteAttribute("Equals() on TensorSpan will always throw an exception. Use the equality operator instead.")]
+#pragma warning disable CS0809 // Obsolete member overrides non-obsolete member
+        public override bool Equals(object? obj) { throw null; }
+#pragma warning restore CS0809 // Obsolete member overrides non-obsolete member
+        public void Fill(T value) { }
+        public System.Numerics.Tensors.TensorSpan<T>.Enumerator GetEnumerator() { throw null; }
+        [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
+        [System.ObsoleteAttribute("GetHashCode() on TensorSpan will always throw an exception.")]
+#pragma warning disable CS0809 // Obsolete member overrides non-obsolete member
+        public override int GetHashCode() { throw null; }
+#pragma warning restore CS0809 // Obsolete member overrides non-obsolete member
+        [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
+        public ref T GetPinnableReference() { throw null; }
+        public static bool operator ==(System.Numerics.Tensors.TensorSpan<T> left, System.Numerics.Tensors.TensorSpan<T> right) { throw null; }
+        public static implicit operator System.Numerics.Tensors.ReadOnlyTensorSpan<T> (System.Numerics.Tensors.TensorSpan<T> span) { throw null; }
+        public static bool operator !=(System.Numerics.Tensors.TensorSpan<T> left, System.Numerics.Tensors.TensorSpan<T> right) { throw null; }
+        public System.Numerics.Tensors.TensorSpan<T> Slice(scoped System.ReadOnlySpan<System.Buffers.NRange> ranges) { throw null; }
+        public T[] ToArray() { throw null; }
+        public override string ToString() { throw null; }
+        public bool TryCopyTo(System.Numerics.Tensors.TensorSpan<T> destination) { throw null; }
+        public ref partial struct Enumerator
+        {
+            private object _dummy;
+            private int _dummyPrimitive;
+            public ref T Current { get { throw null; } }
+            public bool MoveNext() { throw null; }
+        }
+    }
+    public sealed partial class Tensor<T> : System.Collections.Generic.IEnumerable<T>, System.Collections.IEnumerable, System.IEquatable<System.Numerics.Tensors.Tensor<T>>, System.Numerics.IEqualityOperators<System.Numerics.Tensors.Tensor<T>, System.Numerics.Tensors.Tensor<T>, bool>, System.Numerics.Tensors.ITensor<System.Numerics.Tensors.Tensor<T>, T>
```

Comment:
Many of the same comments given above also apply here and to the Tensor/TensorSpanExtensions classes